motor/core.py

```diff
@@ -1120,11 +1119,19 @@ def __init__(self, cursor, collection):
     def __aiter__(self):
         return self
 
-    async def __anext__(self):
-        # An optimization: skip the "await" if possible.
-        if self._buffer_size() or await self.fetch_next:
-            return self.next_object()
-        raise StopAsyncIteration()
+    async def next(self):
```

Comment:
Why add `next`?

Reply:
It allows users to still iterate change streams in a while loop like so:

```
while cursor.alive:
    doc = await cursor.next()
```

Its not a particularly compelling reason, I know but I don't see what harm it does to include it.

Reply:
I guess change stream cursors already have `next()` and `__anext__` too so I guess this is fine. Can you call out the new method in the changelog?

Reply:
Added the changelog entry.